INSTALL.md

```diff
@@ -149,36 +149,36 @@ echo 'extension=oci8.so' > /etc/php.d/oci8.ini
 Confirm it is working with ```php -i | grep -i oci8```
 
 #### MariaDB/MySQL
-The following instructions are to set up a local MYSQL database for GOCDB. They have not been tested in a production enviroment and are currently intended for test instances.
+The following instructions are to set up a local MariaDB/MySQL database for GOCDB. They have not been tested in a production enviroment and are currently intended for test instances.
 
-First you will need to install the MariaDB server and client. Then start the 'mysqld' service.
+First you will need to install the MariaDB/MySQL server and client. Then start the 'mariadb'/'mysqld' service.
```

Comment:
Does MairaDB have its own service? I thought it just called it `mysqld` or is that just an alias?

Reply:
the mariadb package installed on a cloud VM (version 10.1.X) seemingly has its own service for mariadb (and not one for mysql)

```
# systemctl status mysqld
Unit mysqld.service could not be found.
# systemctl status mysql
Unit mysql.service could not be found.
# systemctl status mariadb
   mariadb.service - MariaDB 10.1 database server
   Active: active (running) since Thu 2019-01-17 10:39:49 GMT; 5h 1min ago
```

Reply:
Perhaps this pull request has just been open so long the technology changed?

Reply:
Guess it could be a Red Hat thing.